components/filesystem.rst

```diff
@@ -68,6 +68,10 @@ On POSIX filesystems, directories are created with a default mode value
 
     This function ignores already existing directories.
 
+.. note::
+
+    The directory permissions are affected by the current `umask`_. Set the umask for your webserver, use PHP's :phpfunction:`umask` function or use the :phpfunction:`chmod` function after the directory has been created.
```

Comment:
Please also wrap all lines after the first word that crosses the 72nd character.